src/transaction/estimate_gas.rs

```diff
@@ -3,7 +3,6 @@
 
```

Comment:
I suspect that the `estimate_gas` module can be removed now that the `to` parameter is optional. I'd also be OK if you prefer to do it in a follow up PR.